core/src/test/scala/com/raphtory/api/analysis/graphview/RepeatedEdgeTest.scala

```diff
@@ -22,11 +20,12 @@ class RepeatedEdgeTest extends BaseCorrectnessTest {
   val repeatedInput: Seq[String] = (0 until 4).flatMap(j => input.map(s => s"$s,$j"))
 
   override def setSpout(): Spout[String] = SequenceSpout(repeatedInput: _*)
-
-  override def setGraphBuilder(): GraphBuilder[String] = WeightedGraphBuilder
+  def setGraphBuilder(): GraphBuilder[String] = WeightedGraphBuilder
 
   test("Multilayer edgelist test with repeated edges") {
     val res = repeatedInput.map(s => s"${edges.size - 1},$s")
     correctnessTest(TestQuery(TemporalEdgeList("weight"), edges.size - 1), res)
   }
+
+  override def setSource(): Source = Source(setSpout(), setGraphBuilder())
```

Comment:
No reason to have a separate method for specifying the GraphBuilder, just add it directly to the source